build.sbt

```diff
@@ -88,7 +88,16 @@ lazy val utilDependencies = Seq(
   "org.slf4j" % "slf4j-api" % "1.7.25",
   "org.apache.commons" % "commons-compress" % "1.15",
   "org.tensorflow" % "tensorflow" % "1.8.0",
-  "com.amazonaws" % "aws-java-sdk-s3" % "1.11.313"
+  // Enable the following for tensorflow support
+  //"org.tensorflow" % "libtensorflow" % "1.8.0",
```

Comment:
How did we manage to get rid of TF dependency here? Where are we getting that dependency from now?

Reply:
I didn't, it's two lines above. The libtensorflow and libtensorflow-gpu are required for gpu usage